### PR TITLE
use Git/git instead of Mercurial/hg

### PR DESCRIPTION
**Pull request by matrixise**
Please, could you check this branch ? this branch is not complete because some files ask more work.

### CODE REVIEW

**Comment by matrixise** [comment]
work on `communication.rst`, `devcycle.rst`, `documenting.rst`, `grammar.rst`, `index.rst` and `setup.rst`

**Comment by matrixise** [comment]
https://github.com/python/devguide/issues/46 and https://github.com/python/devguide/issues/50

**Comment by brettcannon** [comment]
Can you do this against the `github` branch?

**Comment by matrixise** [comment]
yep, I will propose to the `github` branch

**Comment by brettcannon** [comment]
You can try changing the base branch directly in this PR, but I didn't want to do it for you in case there were conflicts.

**Comment by matrixise** [comment]
I think there will be some conflicts, because there is a diff between master and github

File: setup.rst [hunk 1]
```diff
@@ -24,12 +24,12 @@ Getting Set Up
...
-CPython is developed using `Mercurial <http://hg-scm.org/>`_. The Mercurial
-command line program is named ``hg``; this is also used to refer to Mercurial
-itself. Mercurial is easily available for common Unix systems by way of the
-standard package manager; under Windows, you might want to use the
-`TortoiseHg <http://tortoisehg.org/>`_ graphical client, but the build system
-still prefers :file:`hg.exe` to be on your PATH.
+CPython is developed using `Git <https://git-scm.com/>`_. The Git command line
+program is named ``git``; this is also used to refer to Git itself. Git is
+easily available for common Unix systems by way of the standard package manager;
+under Windows, you might want to use the `TortoiseGit <http://tortoisegit.org/>`_
```
Comment (brettcannon) [inline setup.rst]: It might be worth also linking to https://desktop.github.com/ as well at TortoiseGit.

Comment (matrixise) [inline setup.rst]: sure, we have some possibilities for a desktop interface for Git.

**Comment by brettcannon** [comment]
@matrixise did you want to resolve the conflicts in this PR? Otherwise I can revert the changes in setup.rst so I can merge this PR.

**Comment by brettcannon** [comment]
I forked this PR so that the work @matrixise did could be used to close out the last GitHub migration milestone bug!

**Pull request closed without merging**